src/script/common/c_content.cpp

```diff
@@ -464,6 +487,133 @@ ContentFeatures read_content_features(lua_State *L, int index)
 }
 
 /******************************************************************************/
+void push_content_features(lua_State *L, const ContentFeatures &f)
+{
+	lua_newtable(L);
+	// TODO
+	/* Name */
+	lua_pushstring(L, f.name.c_str());
+	lua_setfield(L, -2, "name");
+
+	/* Groups */
+	push_groups(L, f.groups);
+	lua_setfield(L, -2, "groups");
+
+	/* Visual definition */
+	setenumfield(L, -1, "drawtype", ScriptApiNode::es_DrawType, f.drawtype);
+	setfloatfield(L, -1, "visual_scale", f.visual_scale);
+
+	lua_createtable(L, 6, 0);
+	for (int i=0; i<6; i++){
+		lua_pushinteger(L, i+1);
+		push_tiledef(L, f.tiledef[i]);
+		lua_settable(L, -3);
+	}
+	lua_setfield(L, -2, "tiles");
+	
+	lua_createtable(L, CF_SPECIAL_COUNT, 0);
+	for (int i=0; i<CF_SPECIAL_COUNT; i++){
+		lua_pushinteger(L, i+1);
+		push_tiledef(L, f.tiledef[i]);
+		lua_settable(L, -3);
+	}
+	lua_setfield(L, -2, "special_tiles");
+	
+	setintfield(L, -1, "alpha", f.alpha);
+	setboolfield(L, -1, "use_texture_alpha", (f.alpha == 0));
+	
+#if 0
```

Comment:
No, that is because that code is still WIP; that is the copy-pasted stuff from read_content_features that still needs to be converted to push things instead of reading them.